### ISSUE TITLE
Could you  provide the prepocessing code of IEMOCAP dataset ?

### ISSUE THREAD

**Issue by hecongqing** [SenticNet/conv-emotion#12]

**Comment by ErickCosta** [comment]
I wanted understanding about the process to generate de IEMOCAP features. This because I want sent new informations for new predictions using the Dialogue RNN model. Other question is aboute the pickle file for IEMOCAP dataset, it is used full dataset to provide file?

**Comment by LaVine-WU** [comment]
Hi

I read your papers and code and it helped me a lot. Can you provide IECOMAP preprocessing code?
Thank  you!

**Comment by lsy641** [comment]
Me too,please share it.

**Comment by devamanyu** [comment]
Hi All, 

We are presently updating our codes to incorporate BERT-based feature encoders in the existing models. We shall upload the extractors soon. 

Meanwhile, please note that the exact models for feature extraction are mentioned in the papers: [text](https://github.com/dennybritz/cnn-text-classification-tf), [openSMILE](https://www.audeering.com/opensmile/) IS13 ComParE, and 3D-CNN.

**Comment by lsy641** [comment]
> Hi All,
> 
> We are presently updating our codes to incorporate BERT-based feature encoders in the existing models. We shall upload the extractors soon.
> 
> Meanwhile, please note that the exact models for feature extraction are mentioned in the papers: [text](https://github.com/dennybritz/cnn-text-classification-tf), [openSMILE](https://www.audeering.com/opensmile/) IS13 ComParE, and 3D-CNN.

I've open the link you gave above, "text" is a piece of code for text binary classsification . So how did you use this to get utterance embbeding in your experiment? was it based on a fine-trained text classifition model? if it was , which dataset  did you use to train this feature extractor?

**Comment by soujanyaporia** [comment]
You can use the same dataset (iemocap if you experiment on iemocap) to pre-train.

**Comment by lrh000** [comment]
> Hi All,
> 
> We are presently updating our codes to incorporate BERT-based feature encoders in the existing models. We shall upload the extractors soon.
> 
> Meanwhile, please note that the exact models for feature extraction are mentioned in the papers: [text](https://github.com/dennybritz/cnn-text-classification-tf), [openSMILE](https://www.audeering.com/opensmile/) IS13 ComParE, and 3D-CNN.

What is the objective (loss) of pre-training the audio and visual features with ComparE or 3D-CNN? Thanks!
